test/events/objectEvent.spec.js

```diff
@@ -131,6 +132,84 @@ describe('unit tests for the ObjectEvent class', () => {
     expect(obj.getEventID().startsWith('ni:///')).to.be.equal(true);
   });
 
+  it('should generate a hashed ID from a string as context', async () => {
+    const oe = new ObjectEvent();
+    oe.setAction(cbv.actionTypes.delete)
+      .addEPCList(exampleObjectEvent.epcList);
+    assert.doesNotThrow(() => { oe.generateHashID('https://gs1.github.io/EPCIS/epcis-context.jsonld'); });
+    expect(oe.getEventID().startsWith('ni:///')).to.be.equal(true);
+    expect(oe.isValid()).to.be.equal(true);
+  });
+
+  it('should generate a hashed ID from an array of string as context', async () => {
+    const oe = new ObjectEvent();
+    oe.setAction(cbv.actionTypes.delete)
+      .addEPCList(exampleObjectEvent.epcList);
+    assert.doesNotThrow(() => {
+      oe.generateHashID([
+        'https://gs1.github.io/EPCIS/epcis-context.jsonld',
+        'https://gs1.github.io/EPCIS/epcis-context.jsonld1',
+        'https://gs1.github.io/EPCIS/epcis-context.jsonld2',
+      ]);
+    });
+    expect(oe.getEventID().startsWith('ni:///')).to.be.equal(true);
+    expect(oe.isValid()).to.be.equal(true);
+  });
+
+  it('should generate a hashed ID from an array of Object as context', async () => {
+    const oe = new ObjectEvent(exampleObjectEvent);
```

Comment:
to be sure that the context is handled correctly, I would add an extension in the objectEvent called ext1, one called ext2, etc.. 
And same for the tests below